pages/_app.tsx

```diff
@@ -2,13 +2,11 @@ import React, { ReactElement, ReactNode, useEffect, useMemo } from 'react';
 import type { AppProps } from 'next/app';
 import 'react-toastify/dist/ReactToastify.css';
 import '@/styles/globals.less';
-import '@dialectlabs/react-ui/index.css';
-import { useRouter } from 'next/router';
+// ts/next fails to find this css module using import
```

Comment:
this feels like a hacky workaround, but I couldnt get anything else to work... any ideas?

Reply:
could it be that the package @dialectlabs/react-ui isn't installed somehow?

Reply:
It's definitely there, including the required .css file. Discussions I found online suggest it's because ts doesnt include css modules/files by default, but messing around with those settings didnt help